CONTRIBUTING.rst

```diff
@@ -77,10 +77,11 @@ Ready to contribute? Here's how to set up `git_wrapper` for local development.
    Now you can make your changes locally.
 
 5. When you're done making changes, check that your changes pass flake8 and the
-   tests, including testing other Python versions with tox::
+   tests, including testing other Python versions with tox, and running integration tests::
 
     $ make lint
     $ make test
+    $ make -eintegration_podman
```

Comment:
I think there should be a space after '-e', but even with that I get:
make: *** No rule to make target 'integration_podman'.  Stop.

Reply:
You're right the command should be tox -eintegration_podman, which needs to be run in the pipenv. I'll update it to show that.